package.json

```diff
@@ -1213,6 +1213,16 @@
                 "command": "vscode-neovim.complete-selection-cmdline",
                 "key": "tab",
                 "when": "neovim.mode == cmdline_normal || neovim.mode == cmdline_insert || neovim.mode == cmdline_replace"
+            },
+            {
+                "key": "ctrl+n",
```

Comment:
is this analogous to `]q` from https://github.com/tpope/vim-unimpaired ? vim-unimpaired is a very well-established convention so I suggest we use its mappings, i.e. `]q` and `[q`.

Reply:
No, this is for navigating quick pick (code actions mostly). However, connecting unimpaired bindings to vscode like `]q` to jump between LSP errors would be a great addition!